Docs: Update publish a plugin FAQ to cover more common questions (#61274)
grafana/grafana: docs/sources/developers/plugins/publish-a-plugin.md

- No. We cannot guarantee specific publishing dates, as plugins are immediately published after a review based on our internal prioritization.

**Can I see metrics of my plugin installs, downloads or usage?**

> - No. We don't offer this information. You are also not allowed to include any kind of tracking code inside your plugin.

Reviewer: ```suggestion
- No. We don't offer this information at the moment to plugin authors.
```
